documentation/SupportingDocuments/API_Documentation/Traffic_Influence_API_Documentation.md

```diff
@@ -117,31 +120,32 @@ Creates a new TrafficInfluence resource
 
 ##### Description
 
-Gets in input an object containing the intents from the AF and creates a TrafficInfluence resourse accordingly. The trafficInfluenceID parameter, that is part of the object, must not be valorized when creating a new resource. It is automatically generated by the system and returned the response
+Gets in input an object containing the intents from the API Consumer and creates a TrafficInfluence resourse accordingly. The trafficInfluenceID parameter, that is part of the object, must not be valorized when creating a new resource. For this reason the trafficInfluenceID parameter must be avoided in the object, anyway it will be ignored by the API Producer. It is automatically generated by the system and returned in the response.
```

Comment:
Please clarify what is meant by 'valorized'? May be clearer  just to say it will be ignored.